integration/spark/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonDropTableCommand.scala

```diff
@@ -82,8 +82,9 @@ case class CarbonDropTableCommand(
         }
       }
 
-      if (SegmentStatusManager.isLoadInProgressInTable(carbonTable)) {
-        throw new ConcurrentOperationException(carbonTable, "loading", "drop table")
+      if (SegmentStatusManager.isInsertInProgress(carbonTable)) {
+        throw new ConcurrentOperationException(carbonTable,
+          "insert or insert overwrite", "drop table")
```

Comment:
the load dml also comes to this code,  for example if user is running "load data inpath XXX into YYY ..." a drop table command comes, he will get the exceptiont which telling 'the table is in insert or insert overwrite'

Reply:
you are right, I have changed